src/server.rs

```diff
@@ -452,6 +452,12 @@ async fn index(
 
     response.headers.extend(&global_response_headers);
 
+    if req.uri().path().eq("/docs") {
+        response
+            .headers
+            .append("Content-Type".to_string(), "text/html".to_string());
+    }
+
```

Comment:
@VishnuSanal , how does this work in case of a const request?

Reply:
Didn't get you, `/docs` was a const request, BTW.

Reply:
Exactly. So, how are handling that case?

Reply:
I still didn't get your question, I am afraid. `/docs` was a const request & this works perfectly fine. plus, nobody can alter the docs page behavior too.